src/cli/cli.cpp

```diff
@@ -2105,8 +2105,14 @@ otError Interpreter::ProcessIpMulticastAddrAdd(Arg aArgs[])
 {
     otError      error;
     otIp6Address address;
+    otIp6Address address2;
 
     SuccessOrExit(error = aArgs[0].ParseAsIp6Address(address));
+    if (!aArgs[1].IsEmpty())
+    {
+        SuccessOrExit(error = aArgs[1].ParseAsIp6Address(address2));
+        SuccessOrExit(error = otIp6SubscribeMulticastAddress(GetInstancePtr(), &address2));
```

Comment:
why not subscribe to the first address first?

Reply:
That would probably be more logical. Because it is a best-effort registration method as currently coded; e.g. if only one slot available then the first address registration will succeed and the 2nd one not.

Reply:
However, the code does not scale to N addresses.

Reply:
Code is now updated to handle 1 or more addresses. (0 is handled as an argument-error.)